devenv/dev-dashboards/datasource_tests_postgres_fakedata.json

```diff
@@ -75,7 +75,7 @@
         {
           "alias": "",
           "format": "time_series",
-          "rawSql": "SELECT\n  $__timeGroup(\"createdAt\",'$summarize'),\n  min(value) as \"value\",\n  'total avg' as \"metric\"\nFROM \n  grafana_metric\nWHERE\n  $__timeFilter(\"createdAt\") AND\n  measurement = 'logins.count'\nGROUP BY time",
+          "rawSql": "SELECT\n  $__timeGroup(\"createdAt\",'$summarize'),\n  min(value) as \"value\",\n  'total avg' as \"metric\"\nFROM \n  grafana_metric\nWHERE\n  $__timeFilter(\"createdAt\") AND\n  measurement = 'logins.count'\nGROUP BY time\nORDER BY time",
```

Comment:
This is only fix here. I assume it got formatted by prettier automatically on commit.

Reply:
Hmm. `\nORDER BY time` was not included before so it can't have been added by prettier right?

Reply:
Yes, I meant this line is the only one change I did (otherwise the graph does not make sense). Other changes are the prettier formatting.